### PR TITLE
fixed issue with lighthouse

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [X] Testing
  - [X] Code is tested

#### What are the relevant tickets?
No tickets

#### What's this PR do?
Fixes url for lighthouse

#### How should this be manually tested?
verify lighthouse stats are shown

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -145,7 +145,7 @@
     "videojs-transcript-ac": "^0.8.8",
     "videojs-youtube": "^2.6.1",
     "walk": "^2.3.14",
-    "webpack": "^4.44.0",
+    "webpack": "^5.73.0",
```
Comment (gumaerc) [inline package.json:148]: Why are we upgrading webpack in this PR?  Did you base your initial branch off of https://github.com/mitodl/ocw-hugo-themes/pull/787?  This PR should be based off of `main` and should only make the change that you made in `.github/workflows/deploy.yml`.

**Review by gumaerc — CHANGES_REQUESTED** [review]
There seems to be an unrelated change in this PR, might be best to rebase it off of `main`:

**Comment by Wassaf-Shahzad** [comment]
> There seems to be an unrelated change in this PR, might be best to rebase it off of `main`:

Yeah it had some unwanted commits I have removed those and is ready to merge

**Review by gumaerc — APPROVED** [review]
:+1: LGTM

**Pull request merged**